src/test/scala/org/broadinstitute/dsde/workbench/avram/CommonTestData.scala

```diff
@@ -14,6 +17,11 @@ import scala.concurrent.ExecutionContext
 
 trait CommonTestData { this: ScalaFutures =>
 
+  val configFactory = ConfigFactory.parseResources("app.conf").withFallback(ConfigFactory.load())
```

Comment:
I think all of this will go away if we move things around a little as I suggested above. That way there isn't any chance that config loading will diverge between main and test.